transports/tcp/Cargo.toml

```diff
@@ -11,26 +11,26 @@ keywords = ["peer-to-peer", "libp2p", "networking"]
 categories = ["network-programming", "asynchronous"]
 
 [dependencies]
-async-io-crate = { package = "async-io", version = "1.2.0", optional = true }
+async-io = { version = "1.2.0", optional = true }
 futures = "0.3.8"
 futures-timer = "3.0"
-if-watch = "2.0.0"
+if-watch = "3.0.0"
 libc = "0.2.80"
 libp2p-core = { version = "0.38.0", path = "../../core" }
 log = "0.4.11"
 socket2 = { version = "0.4.0", features = ["all"] }
-tokio-crate = { package = "tokio", version = "1.19.0", default-features = false, features = ["net"], optional = true }
+tokio = { version = "1.19.0", default-features = false, features = ["net"], optional = true }
 
 [features]
-tokio = ["tokio-crate"]
-async-io = ["async-io-crate"]
+tokio = ["dep:tokio", "if-watch/tokio"]
+async-io = ["dep:async-io", "if-watch/smol"]
```

Comment:
switching to `dep:tokio` style as was done in other crates — let me know if this one was deliberately left unchanged